example/CMakeLists.txt

```diff
@@ -7,4 +7,5 @@
 # Official repository: https://github.com/CPPAlliance/http_io
 #
 
+add_subdirectory (client)
```

Comment:
`flex` or `crul` please. And we should distinguish server from client examples using this directory structure:
```
example/
    server/
        async_http_server/
    client/
        flex_coro/

Reply:
I've made the suggested changes to the client directory. I used the `http_io_example_client` prefix for CMake targets and no prefix for B2 targets. I'll update the server example accordingly if this looks good.

Reply:
I've made the suggested changes to the client directory. I used the `http_io_example_client` prefix for CMake targets and no prefix for B2 targets. I'll update the server example accordingly if this looks good.